electron-app.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/main/main.js b/main/main.js
+index 3d791c3f39c6f001b8ae92881bfefc181731c8ee..76cff7a239d6e777daa23c7ef2588774c9550c60 100644
+--- a/main/main.js
++++ b/main/main.js
+@@ -18,8 +18,10 @@ function createWindow () {
+     }
+   })
+   mainWindow.isElectron = true;
++  mainWindow.removeMenu();
+   // and load the index.html of the app.
+-  mainWindow.loadFile('musicblocks/index.html');
++  mainWindow.loadFile('index.html');
++/*
+   mainWindow.on('close', function(e){
+     var choice = require('electron').dialog.showMessageBoxSync(this,
+         {
+@@ -32,6 +34,7 @@ function createWindow () {
+          e.preventDefault();
+        }
+   });
++*/
+ 
+   // Open the DevTools.
+   // mainWindow.webContents.openDevTools()
```

Comment:
Do you still need this patch ?

Reply:
Yes – the `main.js` from `musicblocks-app` has the wrong filepath for `index.html` and there's also a bug with the prompt that's given when the user tries to exit the app which makes it impossible to quit the app without force-quitting.